cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,121 @@
+@dataclasses.dataclass
+class IcebergAddress:
+    path: str
+    catalog: str
+    table: str
+
+    @classmethod
+    def from_url(cls, url: str):
+        iceberg_url = URL(url)
+        if iceberg_url.host == ".":
+            iceberg_url.path = iceberg_url.path.lstrip("/")
+        return cls(path=iceberg_url.path, catalog=iceberg_url.query_params.get("catalog"), table=iceberg_url.query_params.get("table"))
```

Ruff (0.11.9)

Line too long (135 > 120)

(E501)